app/js/lib/foxbox/event-dispatcher.js

```diff
@@ -169,7 +157,8 @@ export default class EventDispatcher {
 
   /**
    * Removes all registered listeners for the specified event.
-   * @param {string} eventName Name of the event to remove all listeners for.
+   *
+   * @param {string} [eventName] Name of the event to remove all listeners for.
```

Comment:
nit: Do the brackets around the param name for optional? If we use Closure Compiler flavour of JSDoc, you can indicate optional params with a `=` (see https://developers.google.com/closure/compiler/docs/js-for-compiler#types):
```js
/**
 * Removes all registered listeners for the specified event.
 *
 * @param {string=} eventName Name of the event to remove all listeners for.
 */
```